js/ui/map/provider.dynamic.google.js

```diff
@@ -264,6 +265,14 @@ var GoogleProvider = DynamicProvider.inherit({
         return Promise.resolve();
     },
 
+    isEventsCanceled: function() {
+        var gestureHandling = this._map.get("gestureHandling");
```

Comment:
We should check for the map instance presence as it may not be been initialized at this time.

Reply:
I fixed this in the following commit.